digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala

```diff
@@ -3142,7 +3141,130 @@ class ProjectServiceSpec extends FunSuite with Matchers with BeforeAndAfter {
     }
   }
 
-  // note, functionality moved here from former ViiteTierekisteriClient at Tierekisteri removal (2021-09)
+  test("Test defaultSectionCalculatorStrategy.updateRoadwaysAndLinearLocationsWithProjectLinks() " +
+       "When two track road with a new track 2 and partially new track 1 having new track at the end" +
+       "Then roadways should be formed correctly.") {
```

Comment:
"Then roadways should be formed correctly."
Aukipurku: mitä on "correctly" tässä?